src/components/Presentation/SlideNavigator/index.jsx

```diff
@@ -35,6 +40,21 @@ function SlideNavigator({ slides }) {
     },
   });
 
+  const updateSlideOrderMutation = useMutation(
+    async ({ newOrder }) => {
+      const response = await axiosInstance.put(
+        `/users/${userId}/presentations/${presentationId}/slides`,
+        { newOrder },
+      );
+      return response.data;
+    },
+    {
+      onSuccess: () => {
+        updateSlideOrderMutation.invalidateQueries();
+      },
+    },
+  );
```

Comment:
```suggestion
      const response = await axiosInstance.put({
          mutationFn:  async ({ newOrder }) => {
      const response = await axiosInstance.put(
        `/users/${userId}/presentations/${presentationId}/slides`,
        { newOrder },
      );
      return response.data;
    },
    // ...
      });
```
mutationFn 프로퍼티에 fetch하는 함수를 콜백으로 넣는 걸로 통일해야 될 것 같습니다 🙇‍♂️